spec/std/big/big_rational_spec.cr

```diff
@@ -149,7 +149,7 @@ describe BigRational do
   it "#hash" do
     b = br(10, 3)
     hash = b.hash
-    hash.should eq(b.to_f64.hash)
+    hash.should eq(b.to_big_f.hash)
```

Comment:
Float64 has only 53bit meaningful mantisa, but for hash calculation I use 61bit Mersen prime.
That is why conversion to Float64 looses precision.

As an alternative, it could stick with 31bit Mersen prime. Then here could be `to_f64`.
But then Int32, UInt32, Float64 hashing will have to perform modulo operation.

I believe, hashing Int32, UInt32 is much more important, than hashing BigRational and BigFloat.
That is why I decided to use 61bit Mersen Prime, and fallback to `to_big_f` at this place, and to calculate BigFloat hash by algorithm, instead of using `to_f64`.

It is just tradeoff. If you will decide, that BigRational and BigFloat is more important, than Float64 and Int32, I will change it back.